UCTSearch.cpp

```diff
@@ -918,8 +918,8 @@ int UCTSearch::think(int color, passflag_t passflag) {
                 dump_GUI_stats(m_rootstate, m_root);
             }
 
-            keeprunning = (!m_hasrunflag || (*m_runflag));
-            keeprunning &= stop_thinking(centiseconds_elapsed, time_for_move);
+            keeprunning = (!m_hasrunflag || (*m_runflag)) ;
```

Comment:
space before semicolon?

Reply:
That was my mistake. I will fix it late.